src/main/webapp/src/app/index.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'googlemaps';
```

Comment:
Did this wind up doing anything useful? If so, add a comment as to what it's doing, if not remove it.